editor/plugins/skeleton_3d_editor_plugin.cpp

```diff
@@ -287,6 +222,8 @@ void BoneTransformEditor::update_enabled_checkbox() {
 
 void BoneTransformEditor::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_value_changed", "value"), &BoneTransformEditor::_value_changed);
+	ClassDB::bind_method(D_METHOD("_value_changed_vector3", "value"), &BoneTransformEditor::_value_changed_vector3);
```

Comment:
Since you use `callable_mp` which refers to the function pointer directly, you don't need to bind the method. Similarly, all three other pre-existing bindings can be removed (and thus the whole `_bind_methods` override).

Reply:
I removed the `_bind_methods` override since it's no longer needed. :+1: